superset/connectors/sqla/models.py

```diff
@@ -298,20 +300,14 @@ class SqlaTable(Model, BaseDatasource):
         'MAX': sa.func.MAX,
     }
 
-    def get_label(self, label):
-        """Conditionally mutate a label to conform to db engine requirements
-        and store mapping from mutated label to original label
-
-        :param label: original label
-        :return: Either a string or sqlalchemy.sql.elements.quoted_name if required
-        by db engine
-        """
+    def make_sqla_column_compatible(self, sqla_col, label=None):
+        original_label = label if label else sqla_col.name
```

Comment:
Should original_label be renamed to expected_label ?

Reply:
Good point. To align with semantics elsewhere, I changed this to `label_expected`.